src/ToricVarieties/NormalToricVarieties/attributes.jl

```diff
@@ -364,23 +374,51 @@ julia> fan_of_variety(p2)
 A polyhedral fan in ambient dimension 2
 ```
 """
-function fan_of_variety(v::NormalToricVariety)
+function fan_of_variety(v::AbstractNormalToricVariety)
     return PolyhedralFan(pm_object(v))
 end
 export fan_of_variety
 
 
 @doc Markdown.doc"""
-    fan(v::NormalToricVariety)
+    fan(v::AbstractNormalToricVariety)
 
-A convenience method for fan_of_variety of a normal toric variety `v`.
+A convenience method for fan_of_variety of an abstract normal toric variety `v`.
 """
-function fan(v::NormalToricVariety)
+function fan(v::AbstractNormalToricVariety)
     return fan_of_variety(v)
 end
 export fan
 
 
+@doc Markdown.doc"""
+    cone_of_variety(v::AffineNormalToricVariety)
+
+Returns the cone of the affine normal toric variety `v`.
+
+# Examples
+```jdoctest
+julia> cone_of_variety(AffineNormalToricVariety(Oscar.positive_hull([1 1; -1 1])))
+A polyhedral cone in ambient dimension 2
+```
+"""
+function cone_of_variety(v::AffineNormalToricVariety)
+    return maximal_cones(fan(v))[1]
+end
+export cone_of_variety
+
+
+@doc Markdown.doc"""
+    cone(v::AffineNormalToricVariety)
+
+A convenience method for cone_of_variety of an affine toric variety `v`.
```

Comment:
In how far is this a "convenience method"?! Naively, why not just remove `cone_of_variety` and only keep `cone`? That seems like the "Julia-ish" way to do it. Same for `fan` and `fan_of_variety`...

Reply:
Maybe just `PolyhedralFan(...)` and `Cone(...)` so it looks like a constructor? Or would that be bad style?

Reply:
Convenience as in "you need to type less". But I agree that we can as well opt for just one command. I would in this case prefer `Fan` over `PolyhedralFan`, because the former is shorter. I have not preference with regard to capitalization.